tasks/RedHat.yml

```diff
@@ -12,7 +12,7 @@
 - name: Download package locally
   become: no
   connection: local
-  get_url: "url={{ couchbase_server_centos_ee_url }} dest={{ role_path }}/files/{{ couchbase_server_centos_ee_package }} sha256sum={{ couchbase_server_centos_ee_sha256 }} timeout=240"
+  get_url: "url={{ couchbase_server_centos_ee_url }} dest={{ role_path }}/files/{{ couchbase_server_centos_ee_package }} mode=0644 sha256sum={{ couchbase_server_centos_ee_sha256 }} timeout=240"
```

Comment:
The destination is "{{ role_path }}" and it is in the control machine (i.e. the machine that you are using to deploy the changes).
The assumption is that you "control" the "control machine" and you can change those permission.